data.sql

```diff
@@ -10,4 +10,62 @@ insert into animals (id, name, date_of_birth, escape_attempts ,  neutered, weigh
 values (3,  'Pikachu', '2021-01-07', 1, false, 15.0) 
 
 insert into animals (id, name, date_of_birth, escape_attempts ,  neutered, weight_kg )
-values (4,  'Devimon', '2017-06-12', 5, true, 11) 
\ No newline at end of file
+values (4,  'Devimon', '2017-06-12', 5, true, 11) 
+
+insert into animals (id, name, date_of_birth, escape_attempts ,  neutered, weight_kg )
+values 
+(5, 'Charmander', '2020-02-08', 0, false, -11)
+(6, 'Plantmon', '2021-11-15', 2, true, -5.7)
+(7, 'Squirtle', '1993-04-02', 3, true, -12.13) , 
+(8, 'Angemon', '2005-06-12', 1, true, -45) ,
+(9, 'Boarmon', '2005--07-07', 7, true, 20.4) ,
+(10, 'Blossom', '1998-10-13', 3, true, 17) ,
+(11, 'Ditto', '2022-05-14', 4, true, 22)
+
+
+insert into owners (full_name, age)
```

Comment:
Great job @r-ahmed2022 🤝

- [ ] As stated in the project requirements, kindly add all the data to the `owners` table.
![melody](https://user-images.githubusercontent.com/63917725/220841258-1d7e8b3a-99cb-4204-bdb3-fc72f2fd53c8.PNG)

Reply:
done, as advised